### ISSUE TITLE
Content: Improve "Stay home" page intro

### ISSUE THREAD

**Issue by coreycaitlin** [cagov/covid19#334]
As we get deeper into the Stay Home order (and further from the immediate announcements that put it in place), I think the Stay Home page needs to offer a clearer, more directive, quick-hit summary of what we're telling people to do.

This has two levels:

1. What's in the executive order:
    - Stay home
    - Essential needs
    - Practice social distancing
2. Softer recommendations (obviously with a different header)
    - Wash your hands
    - Cover your face
    - Clean stuff

In other words, this page (instead of the homepage) should be the single source that tells me (a Californian who is not sick or looking for help of any kind) what I should do.

This can be iterative, and I think clarifying the top of the page should be the priority — once that's done, we should move the remaining content out of FAQ, and into more direct and skimmable language.

**Comment by KaylaPrusia** [comment]
Stay at Home FAQ staging doc: https://docs.google.com/document/d/18MbtHE59SOXS7FDzGXfFdA5v7NbsJklc/edit

**Comment by KaylaPrusia** [comment]
@abquirarte @coreycaitlin , @thepegisin and I have reviewed and made suggestions to the FAQ: https://docs.google.com/document/d/18MbtHE59SOXS7FDzGXfFdA5v7NbsJklc/edit#

Note: Last section "Government Services" is based off of moving those 2 questions from Public Places page and are not yet live on the site.

**Comment by abquirarte** [comment]
Once approved:

- [ ] retire public places page. Re-route to stay home FAQ

**Comment by abquirarte** [comment]
This is complete. Only thing left is Public Places
